internal/cmd/test-bridged-tap/main.go

```diff
@@ -28,7 +28,7 @@ import (
 
 	"github.com/containernetworking/cni/pkg/skel"
 	"github.com/containernetworking/cni/pkg/types"
-	"github.com/containernetworking/cni/pkg/types/current"
+	current "github.com/containernetworking/cni/pkg/types/040"
```

Comment:
I think 100 is current.

Reply:
i chose `040` because it won't break the build. `100` requires more code changes. i take a look into it.

Reply:
fixed.